pandas/_testing.py

```diff
@@ -1104,6 +1105,8 @@ def assert_series_equal(
         Compare datetime-like which is comparable ignoring dtype.
     check_categorical : bool, default True
         Whether to compare internal Categorical exactly.
+    check_category_order : bool, default True
```

Comment:
can we simply enforce ordering when check_categorical is True? alteratively, could add 'exact' as a possiblity in check_categorical (just see what is easier here, not averse to adding another arg)

Reply:
I think adding the extra arg is easiest. `pytest.mark.parametrize` doesn't work with optional arguments so there's no way to use optional arguments to cut down on args needed. Adding a string possibility to a boolean argument isn't very intuitive (and changing `check_categorical` to a string argument would result in a lot of changes in other files)